src/lib/errors/codes.ts

```diff
@@ -0,0 +1,34 @@
+import { ErrorCode as SDKErrorCode } from '@jolocom/sdk'
+export { SDKErrorCode }
+
+export enum ErrorCode {
```

Comment:
they should be separate because they have different domains (SDK errors vs. Application layer errors that are more about the actual UX)
But SDK errors are just a copy of the App errors currently, the App Errors need to be simplified (after having been basically "moved" to the SDK)